src/integrationtests/java/com/aws/greengrass/integrationtests/NucleusLaunchUtils.java

```diff
@@ -52,18 +54,72 @@
     @Mock
     ShadowManagerDatabase mockShadowManagerDatabase;
 
+    @Deprecated()
```

Comment:
Are we not cleaning these up because they are being used across all the unit tests?

Reply:
Yes. I will clean them up in a later PR since these are used in all the integration tests. Didn't want to make the PR any bigger.